app/services/webhooks/invoices/created_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Webhooks
+  module Invoices
+    class CreatedService < Webhooks::BaseService
```

Comment:
Are we sure all these services are _really_ useful? 🤔

Reply:
well, that's a good question, I would say yes because we override some values (ei: `webhook_type`, `serializer`) but maybe we can think about an other way to do it